clarifai/cli/model_templates.py

```diff
@@ -0,0 +1,182 @@
+"""Templates for model initialization."""
+
+from clarifai import __version__
+
+
+def get_model_class_template() -> str:
+    """Return the template for a basic ModelClass-based model."""
+    return '''from typing import Iterator
+
+from clarifai.runners.models.model_class import ModelClass
+from clarifai.runners.utils.data_types import Text
+
+
+class MyModel(ModelClass):
+    """A custom model implementation using ModelClass."""
+
+    def load_model(self):
+        """Load the model here.
+        
+        # TODO: please fill in
+        # Add your model loading logic here
+        """
+        pass
+
+    @ModelClass.method
+    def predict(self, prompt: Text = "") -> Text:
```

Comment:
@srikanthbachala20 let's use str -> str here so it's not something else for a user to learn to import Text.